bin/clover/src/pipeline-steps/assetSpecificOverrides.ts

```diff
@@ -104,5 +105,95 @@ const overrides = new Map<string, OverrideFn>([
     setAnnotationOnSocket(socket, { tokens: ["Id"] });
 
     variant.sockets.push(socket);
-  }]
+  }],
+  [
+    "AWS::SecretsManager::Secret",
+    addSecretProp("Secret String", "secretString", ["SecretString"]),
+  ],
 ]);
+
+function addSecretProp(
+  secretKind: string,
+  secretKey: string,
+  propPath: string[],
+) {
+  return (spec: ExpandedPkgSpec) => {
+    const variant = spec.schemas[0].variants[0];
+
+    const [secretName] = propPath.slice(-1);
+    if (!secretName) {
+      return;
+    }
+
+    // Find secret prop
+    let secretParent = variant.domain;
+    let secretProp: ExpandedPropSpec | undefined = variant.domain;
+
+    for (const propName of propPath) {
+      // If we haven't found the secret prop yet, and we're not with an object in hand, break
+      if (secretProp.kind !== "object") {
+        secretProp = undefined;
+        break;
+      }
+
+      secretParent = secretProp;
+      const thisProp = secretParent.entries.find((p) => p.name === propName);
+
+      // If we don't find the prop on the parent, break
+      if (!thisProp) {
+        secretProp = undefined;
+        break;
+      }
+
+      secretProp = thisProp;
+    }
+
+    if (!secretProp) {
+      console.log(`Could not add secret value for ${spec.name}`);
+      return;
+    }
+
+    // Find propUsageMap
```

Comment:
Not at all needed right now, but it feels like we're begging for a getPropFromPropPath() method here!